sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodOverloadsShouldBeGrouped.cs

```diff
@@ -151,19 +151,39 @@ class J
     // https://github.com/SonarSource/sonar-dotnet/issues/2776
     public class StaticMethodsTogether
     {
```

Comment:
would it be useful to also add tests with fields and properties?

Reply:
The rule is not implemented for fields and properties due to RSPEC:
> Method overloads should be grouped together

Interleaving by fields is tested above.